### PR TITLE
Replace deprecated kafka library

### PR DESCRIPTION
**Pull request by DrmagicE**
related to #324

### CODE REVIEW

File: broker/kafka/context.go [hunk 1]
```diff
@@ -2,7 +2,6 @@ package kafka
 
 import (
 	"context"
-
```
Comment (asim) [inline broker/kafka/context.go]: Do not strip this newline. We like to separate stdlibs from other urls.

File: broker/kafka/kafka.go [hunk 2]
```diff
@@ -3,39 +3,39 @@ package kafka
 
 import (
 	"context"
-	"sync"
-
```
Comment (asim) [inline broker/kafka/kafka.go]: Don't strip this newline

File: broker/kafka/kafka.go [hunk 3]
```diff
@@ -3,39 +3,39 @@ package kafka
...
 	"github.com/Shopify/sarama"
 	"github.com/google/uuid"
 	"github.com/micro/go-log"
 	"github.com/micro/go-micro/broker"
 	"github.com/micro/go-micro/cmd"
 	"github.com/micro/go-micro/codec/json"
-	sc "gopkg.in/bsm/sarama-cluster.v2"
+	"sync"
```
Comment (asim) [inline broker/kafka/kafka.go]: Move sync to be with the stdlib packages

**Comment by DrmagicE** [comment]
Fixed. Please check it, thanks.

**Pull request merged**